frontend/src/Components/Player.jsx

```diff
@@ -100,8 +100,10 @@ export default function Player({ playerName, style, childRef }) {
 
     return (
         <div className={playerStyle} style={style} ref={childRef} onClick={validateOnClick(onClick)}>
+            {isHoveredPlayerDead ? <img src={avatarImageDead} alt="dead player avatar" className={styles.playerImg}/> : <img src={avatarImage} alt="player avatar" className={styles.playerImg}/>}
             <div className={styles.playerText}>
                 <p>{playerName.concat(isHoveredPlayerDead ? ' (DEAD)' : '')}</p>
+                <p>{isPlayer ? ' (YOU)' : ''}</p>

```

Comment:
This wasn't part of the original design, but I don't mind it, helps make position of character clear.

Reply:
Yeh, I did this as a compromise. The original UI had the player blob a different colour to the rest. As this is no longer done, I instead used text to indicate this.

Reply:
Sorry, I meant the original hifi design in the linked issue

Reply:
I know. The hifi did not have any indication of what player you are. The original UI did, so i carried that feature over in a different way.